frontend/src/types/RegistrationTypes.ts

```diff
@@ -0,0 +1,6 @@
+export type CartItem = {
+  name: string;
+  campers: number;
+  totalPrice: number;
+  details?: string;
```

Comment:
what is the details field for and why is it optional?

Reply:
The "150 minutes" field, but yeah I might've forgotten to render it, will add

<img width="273" alt="Screen Shot 2022-12-14 at 11 19 43 PM" src="https://user-images.githubusercontent.com/40974372/207771655-8f540a82-b04c-4c6d-a035-19958e7bd25d.png">